app/assets/stylesheets/application.scss

```diff
@@ -77,14 +78,45 @@ body {
   height: 142px;
 }
 
-.site-footer {
+footer {
+  background-color: $darkslategrey;
+  bottom: 0;
+  color: #ffffff;
+  font-size: 18px;
+}
+
+// Nav
+.top-bar ul li {
   background: $darkslategrey;
 }
 
+.top-bar a {
+  font-size: 16px;
+}
+
+.name img {
+  background: inherit;
+  margin: 10px;
+  height: 70px;
+  width: auto;
+  padding-left: 10px;
+  padding-bottom: 10px;
+}
+
+.top-bar-section img {
+  background: inherit;
+  float: right;
+  margin-top: 7px;
```

Comment:
Properties should be ordered background, float, margin-right, margin-top, max-height, width